pkg/templates/templates_generated.go

```diff
@@ -4568,6 +4623,43 @@ func linuxCloudInitArtifactsUbuntuCse_install_ubuntuSh() (*asset, error) {
 	return a, nil
 }
 
+var _linuxCloudInitConfigHcpmumanskiebld51438502EastusSvc0 = []byte(`apiVersion: v1
```

Comment:
some make target probably messed up depending on which you ran? these shouldnt be here

Reply:
re-running `make generate` again with a clean branch with only necessary changes, may make it work or at least help check that this was added by the make target.

Reply:
Seems like removing accidentally added files helped with make generate